service/iot-service-samples/digitaltwin-service-samples/thermostat-service-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/Thermostat.java

```diff
@@ -48,14 +48,13 @@ private static void RunSample() {
     }
 
     private static void InitializeServiceClient() {
-        DigitalTwinAsyncClient asyncClient = new DigitalTwinAsyncClient(iotHubConnectionString);
-        client = new DigitalTwinClient(asyncClient);
+        asyncClient = DigitalTwinAsyncClient.createFromConnectionString(iotHubConnectionString);
     }
 
     private static void GetDigitalTwin()
     {
         // Get the digital twin.
-        BasicDigitalTwin getResponse = client.getDigitalTwin(digitalTwinid, BasicDigitalTwin.class);
+        BasicDigitalTwin getResponse = asyncClient.getDigitalTwin(digitalTwinid, BasicDigitalTwin.class).toBlocking().single();
```

Comment:
why call async client and block on it? why not call the sync client directly - like you had it before?

Reply:
As in my PR description, Just to show how to use the async client, I made one sample to use it.

Reply:
In that case, the sample should show how you would call observable APIs and subscribe to them.
I don't think initializing an async client and then blocking on it demonstrates the async client usage properly, right?